src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -88,17 +90,16 @@ const ChatsList: React.FunctionComponent = () => {
 
   const chatsToDisplay = useMemo(
     () =>
-      privateChats &&
       privateChats
-        .filter(
+        ?.filter(
           (message) =>
             message.deleted !== true &&
             message.type === "private" &&
             (message.to === selectedUser?.id ||
               message.from === selectedUser?.id) &&
             message.ts_utc.seconds > HIDE_BEFORE
         )
-        .sort(chatSort),
+        .sort(chatSort) ?? [],
```

Comment:
Another pattern for this (not sure which is better) would be to do `(privateChats ?? [])` up front, and then the rest doesn't need to handle potentially `undefined`

Reply:
That sounds better yep. We could catch this with a linter perhaps?

Reply:
It'd certainly be possible, but would take a little time to figure the semantics of it properly (assuming a rule doesn't already exist)

If you create a backlog ticket for it I can make this happen when we have some breathing room.